README.md

```diff
@@ -119,8 +127,6 @@ to the EC2 instance.
 
 * `efs_file_system` – (Optional) An [efs\_file\_system](#efs_file_system) block used to define the EFS file system to be attached to the EC2 instance(s).
 
-* `eip_create` – (Optional) A Boolean value specifying that an elastic IP address should be created and attached to the EC2 instance(s). Default is false.
```

Comment:
@akibare999 You still want something here that points to the eip block blow. Something like this:

```
* `eip` – (Optional) An [eip](#eip) block used to create or lookup an Elastic IP to attach.
```